siriuspy/siriuspy/csdevice/currinfo.py

```diff
@@ -0,0 +1,91 @@
+"""Define PVs, contants and properties of all CurrInfo SoftIOCs."""
+import collections as _collections
+
+
+OFFONTYP = ('Off', 'On')
+DCCTSELECTIONTYP = ('Avg', 'DCCT13C4', 'DCCT14C4')
+BUFFAUTORSTTYP = ('PVsTrig', 'DCurrCheck', 'Off')
+
+
+class Const:
+    """Const class defining CurrInfo constants and Enum types."""
+
+    @staticmethod
+    def _init():
+        """Create class constants."""
+        for i in range(len(DCCTSELECTIONTYP)):
+            Const._add_const('DCCT', DCCTSELECTIONTYP[i], i)
+        for i in range(len(OFFONTYP)):
+            Const._add_const('DCCTFltCheck', OFFONTYP[i], i)
+        for i in range(len(BUFFAUTORSTTYP)):
+            Const._add_const('BuffAutoRst', BUFFAUTORSTTYP[i], i)
+
+    @staticmethod
+    def _add_const(group, const, i):
+        if not hasattr(Const, group):
+            setattr(Const, group, _collections.namedtuple(group, ''))
+        obj = getattr(Const, group)
+        setattr(obj, const, i)
+
+
+Const._init()  # create class constants
+
+
+def get_charge_database():
+    """Return CurrentInfo-Charge Soft IOC database."""
+    pvs_database = {
+        'Version-Cte':        {'type': 'string', 'value': 'UNDEF'},
```

Comment:
@anacso17 , we have been using <code>'str'</code> for pcaspy string PV type in database entries, whereas you used <code>'string'</code>. do you know if both options are available? maybe we should try to stick with one option.

Reply:
update: PCASpy documentation mention <code>'string'</code> as an option for the PV type, but does not mention <code>'str'</code>. i will create another PR replacing all <code>'str'</code> instances to <code>'string'</code>.